.github/workflows/odh-build-and-publish-operator-image.yaml

```
@@ -0,0 +1,157 @@
+# This is a copy of the publish-core-images.yaml and has been customized to
+# use the quay login credentials and RHOAI Dockerfile.
+# The unused parts of the original have been commented out on purpose.
+name: ODH
```

```suggestion
name: ODH Build and Publish Image
```
a better name can be given